src/Equinor.ProCoSys.BusReceiver/Topics/McPkgTopic.cs

```diff
@@ -4,7 +4,10 @@ public class McPkgTopic
     {
         public string ProjectSchema { get; set; }
         public string ProjectName { get; set; }

```

Comment:
can an MCpkg be moved to another project as well? (either just mc pkg, or the parent comm pkg is moved?)

Reply:
Since a McPkg has a CommPkg as a parent, and CommPkg can be moved between projects, the ProjectName for the McPkg must be updated accordingly

Reply:
If a commpkg is moved => all mc pkgs are moved
Then we should change the project of IPO (since entire scope is moved)